apps/web-portal/lib/middlewares.js

```diff
@@ -0,0 +1,24 @@
+import cors from 'cors'
+import bodyParser from 'body-parser'
+
+export default {
+  json: bodyParser.json({
+    limit: 1024 * 1024,
```

Comment:
make limit configurable

Reply:
I will move the configuration to the app config after the gh-111 will be merged

Reply:
extracted to separate task https://github.com/zorko-io/zorko-platform/issues/161